src/mc_tools.jl

```diff
@@ -10,10 +10,7 @@ References
 
 http://quant-econ.net/jl/finite_markov.html
 =#
-
-# new method to check if all elements of an array x are equal to p
-isapprox(x::Array, p::Number) = all([isapprox(x[i], p) for i=1:length(x)])
-isapprox(p::Number, x::Array) = isapprox(x, p)
+import LightGraphs: DiGraph, period, attracting_components, strongly_connected_components
```

Comment:
My philosophy would be to locate import statements that multiple sections of the library use in `QuantEcon.jl` and locate import statements that only a couple packages use in their own files. 
Since we're not providing any functionality to the user such that they interact with any of the other types or methods in LightGraphs, this is why i chose `import` over `using`.